pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
Ahh I see; I was intending this to be a simple general file sorter not specifically for python and as such should not respect a particular language's commenting. i.e. a deploy-whitelist.txt file I don't think people would expect python comments to be treated differently.